cmd/xgql/main.go

```diff
@@ -210,7 +217,12 @@ func main() {
 		rt.Handle("/", playground.Handler("GraphQL playground", "/query"))
 	}
 
-	h := http.Server{
+	// start health endpoints to aid in routing traffic to the pod
+	if err := startHealth(internal.HealthOptions{Health: *health, HealthPort: *healthPort}, log); err != nil {
+		kingpin.FatalIfError(err, "cannot start health endpoints")
+	}
```

Comment:
nitpick: you can just wrap the `startHealth` call in `kingpin.FatalIfError` if you like